index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Flutter is a mobile app SDK for iOS and Android
+title: Welcome to Flutter
```

Comment:
Updating the title isn't so much about clarifying for people visiting the homepage, it's for SEO. When we update the title, it gives search results a much more clear one-liner. Seeing a one-liner for what Flutter is, is much more helpful than "Welcome to Flutter" in Google search results.